src/main/java/fr/techad/edc/httpd/utils/TokenUtils.java

```diff
@@ -32,11 +40,21 @@ public static synchronized TokenUtils getInstance() {
     return instance;
   }
 
+  public boolean getAndVerifyToken(HttpServerExchange exchange) throws IOException {
```

Comment:
Rename this function because you don't get the token. You get it inside to validate it. So the right name is **validateToken**